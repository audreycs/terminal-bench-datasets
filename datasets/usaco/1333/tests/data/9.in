10
263959270418486736 511944810471618212
719955631781742290 884235488590662642
911958183634412763 115875852572241461
882870749898577341 896059443335458468
497522514232414628 118311430658869194
953631769711592758 442130823895113645
516877924612139121 25792863217482507
670853595357487791 22108287664915026
610788629622193538 221128154162799165
410465679038670103 498400723790707670
