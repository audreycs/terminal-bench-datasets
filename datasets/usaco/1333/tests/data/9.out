503
186
463
679
327
331
229
269
269
1664
